README.md

```diff
@@ -1,2 +1,16 @@
 # cre
+
 Code Reliability Engineering Toolbox
+
+## Get the last comment in Gerrit by ID
+
+This tool is used to extract the information in the last comment of a Gerrit revision. We should pass the server where gerrit is hosted, the ID number of the revision and the path of the SSH key to connect to the given server.
```

Comment:
Actually, if I read this correct, we work with change numbers probably here?
Lets make it clear – in Gerrit terminology the change ID is the string like "Ic8aaa0728a43936cd4c6e1ed590e01ba8f0fbf5b" and the change number is human-readable value like 12345. If it is the latter, I would update and be consistent everywhere to avoid confusion.

Reply:
Yes, you're right. It's explained in the [Gerrit User Guide](https://gerrit-review.googlesource.com/Documentation/intro-user.html). Change done.